index.js

```diff
@@ -47,11 +48,22 @@ class Conf {
 		};
 
 		if (!options.cwd) {
-			if (!options.projectName) {
-				const pkgPath = pkgUp.sync(parentDir);
-				// Can't use `require` because of Webpack being annoying:
-				// https://github.com/webpack/webpack/issues/196
-				options.projectName = pkgPath && JSON.parse(fs.readFileSync(pkgPath, 'utf8')).name;
+			const packageData = this._getPackageData();
```

Comment:
Nifty pick: I would probably like to refactor this to something like `_inferProjectDataFromPackage` to highlight the goal of this function. 

Thoughts on this?

Reply:
I agree. Let’s try it.

Reply:
On a closer look, the function doesn't actually infer the data from the package.json, it just provides it. 
So I guess I'll leave it as it is 🤷‍♂